sph/include/sph/hydro_turb/driver.hpp

```diff
@@ -92,16 +93,17 @@ void updateNoise(std::vector<T>& phases, T stddev, T dt, T ts, std::mt19937& gen
 
 /*! @brief Adds the stirring motion to particle accelerations
  *
- * @tparam Dataset
- * @param  startIndex
- * @param  endIndex
- * @param  d
+ * @tparam Dataset          SPH hydro dataset
+ * @tparam TurbulenceModes  Turbulence stirring modes

```

Comment:
Do you mean turb.modes? I don't see TurbulenceModes anywhere.

Reply:
Left-over template parameter, I'll remove it.